service/controller/resource/prometheus/resource.go

```diff
@@ -177,8 +177,8 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 					corev1.ResourceMemory: *key.PrometheusDefaultMemory(),
 				},
 				Limits: corev1.ResourceList{
-					// cpu: 100m
-					corev1.ResourceCPU: *key.PrometheusDefaultCPU(),
+					// cpu: 150m
+					corev1.ResourceCPU: *key.PrometheusCPULimit(),
```

Comment:
is this the _default_ CPU limit? updating the naming might be helpful

Reply:
I think this would make sense

Reply:
renamed to `PrometheusDefaultCPULimit`